fuzzing/scripts/build/zlib.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Copyright 2018-2019 by
+# Armin Hasitzka.
+#
+# This file is part of the FreeType project, and may only be used, modified,
+# and distributed under the terms of the FreeType project license,
+# LICENSE.TXT.  By continuing to use, modify, or distribute this file you
+# indicate that you have read the license and understand and accept it
+# fully.
+
+dir="${PWD}"
+cd $( dirname $( readlink -f "${0}" ) ) # go to `/fuzzing/scripts/build'
+
+path_to_src=$( readlink -f "../../../external/zlib" )
+# Cannot do out of source build with zprefix, too many zconf.h
+path_to_build="${path_to_src}"
+path_to_install="${path_to_src}/usr"
+
+if [[ "${#}" -lt "1" || "${1}" != "--no-init" ]]; then
+
+    git submodule update --init "${path_to_src}"
+
+    cd "${path_to_src}"
+
+    git clean -dfqx
+    git reset --hard
+    git rev-parse HEAD
+
+    mkdir -p "${path_to_build}" && cd "${path_to_build}"
+    sh ./configure --zprefix --prefix="${path_to_install}"
```

Comment:
Let's add `--static` here since OSS-Fuzz will only ever use the static builds. That way we can avoid mixing up the two artefacts down the road.